### ISSUE TITLE
prefer-includes: should fail on if (foo.indexOf('bar'))

### ISSUE THREAD

**Issue by guoyunhe** [sindresorhus/eslint-plugin-unicorn#1189]
Here is a common error when using indexOf():

```js
if (foo.indexOf('bar')) {
  doSomething();
}
```

Should be:

```js
if (foo.indexOf('bar') > -1) {
  doSomething();
}

// or

if (foo.includes('bar')) {
  doSomething();
}
```

Rule: prefer-includes

**Comment by fisker** [comment]
They are not equal, 

```js
if (foo.indexOf('bar')) {
  doSomething();
}
```

```js
if (!foo.startsWith('bar')) {
  doSomething();
}
```
